lib/slide_hub/batch.rb

```diff
@@ -24,68 +25,105 @@ def self.execute
       end
     end
   end
+end
+
+class BatchProcedure
```

Comment:
__[reek]__
* [InstanceVariableAssumption] assumes too much for instance variable @key [:ref](https://github.com/troessner/reek/blob/master/docs/Instance-Variable-Assumption.md)
* [TooManyInstanceVariables] has at least 6 instance variables [:ref](https://github.com/troessner/reek/blob/master/docs/Too-Many-Instance-Variables.md)

<!-- Comment by [SideCI](https://sideci.com) -->